### DISCUSSION TITLE
Adapting contributing guide  - License mismatch

### DISCUSSION THREAD

**Discussion by JustWaveThings** [club-40/odin-pantry#17] [General]
I believe that TOP's contributing guide  (is their License a part of the curriculum?) falls under a different license than is currently shown in the license.md for this group project.

From TOP's curriculum repo license.md:
> > License
> This curriculum is licensed under the [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License](http://creativecommons.org/licenses/by-nc-sa/4.0/).
> 
> It basically means you can do whatever you want with the material as long as it's for non-commercial use but you've got to attribute the original author ([Erik Trautman](http://github.com/eriktrautman)) and share any derivative works under the same license.

https://github.com/TheOdinProject/curriculum/blob/main/license.md

In order to create a derivative work from the TOP contributing guide, it sounds like we'd need to adopt that license for our guide.  Now, I think we'd be fine having the whole project under that license, but I think another path is to have **just**  the guide under that specific license. 

What are the team's thoughts on adopting the same license that TOP uses for their curriculum?

**Comment by JustWaveThings** [comment]
This repo has a GNU General Public License v3.0  https://github.com/Club-40/TOP-Club-40-Group-Project/blob/e6626ae4735f89e689f0574fb7013f7fd4902baa/LICENSE.

I believe in order to use the TOP contributing guide or a derivative, we'd need to follow their licensing.  I could be wrong.

**Reply by Philip-Clark** [reply]
I'm fine with changing the license. As long as the new one protects our work and keeps it free and available.

**Comment by Lofty-Brambles** [comment]
Do we need to follow TOP's contributing guide as a derivative? I like GPL-v3's straightforward, strict copyleft  policy and I think allowing that for this project would be cool enough.

If you want an example, here's a contributing guide for another GPL-v3 project.
https://github.com/l-zeuch/yagpdb.vim/blob/master/.github/CONTRIBUTING.md

**Reply by Lofty-Brambles** [reply]
If this is something that could do with a PR, I'm glad to do it in a bit @Philip-Clark

**Reply by Philip-Clark** [reply]
It think we were waiting for some opinions before a PR. But I'm good if you want to make on @Lofty-Brambles

**Reply by JustWaveThings** [reply]
Yeah I am good with sticking with GPL-v3! I just didn't want to start the project on the wrong side of a license.  

I don't know if everyone can see it, but I think you can make an issue from this discussion on the right hand side --- 

![image](https://user-images.githubusercontent.com/48169481/230449163-c0db19bf-7cfb-4e80-93a0-a8f25dd2feaa.png)

**Reply by JustWaveThings** [reply]
@Philip-Clark  --- Am I good to get a rough draft going  with the  example @Lofty-Brambles  gave with the same license this repo has --- I'll incorporate these  suggestions implemented as best I am able --- https://github.com/orgs/Club-40/discussions/2#discussioncomment-5539040 -- from  @fcasibu 
and 
https://github.com/orgs/Club-40/discussions/2#discussioncomment-5543097  -- from @crespire 

Or if this is better handled by someone else, that's fine as well!

**Reply by Lofty-Brambles** [reply]
> @Philip-Clark --- Am I good to get a rough draft going with the example @Lofty-Brambles gave with the same license this repo has --- I'll incorporate these suggestions implemented as best I am able --- https://github.com/orgs/Club-40/discussions/2#discussioncomment-5539040 -- from @fcasibu and https://github.com/orgs/Club-40/discussions/2#discussioncomment-5543097 -- from @crespire

@JustWaveThings if you feel like it, please do so! I can be helping out with a reviews, fo-sure!

**Reply by Philip-Clark** [reply]
@JustWaveThings Sounds great. I think you are a good person to do this.

**Reply by Lofty-Brambles** [reply]
Do go through the points here if you need to: https://www.gnu.org/licenses/gpl-3.0.en.html
